inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -744,9 +780,10 @@ void RemovePermutationsNHWCToNCHWPass::run() {
 }
 
 void InsertIdentityLayerPass::run() {
+    bool lowPrecision = getPassManager()->isLowPrecision();
     auto quantized = InferenceEngine::getInjectedData<QuantizedLayerParams>(pLayers->front());
     for (auto & l : *pLayers) {
-        for (auto && prev : getCandidatesForIdentityInsertion(l)) {
+        for (auto && prev : getCandidatesForIdentityInsertion(l, lowPrecision)) {
```

Comment:
Consider to query for:
bool lowPrecision = getPassManager()->isLowPrecision()
inside getCandidatesForIdentityInsertion()

Reply:
It's done in a similar way  it's done in other helper functions in gna_pass_manager, where passmanager is a paramater. I could switch to this to be more aligned.